src/pdb.rs

```diff
@@ -30,29 +34,37 @@ pub fn get_compact_symbol_table(pdb_data: &[u8], breakpad_id: &str) -> Option<Co
             })) => Some((offset.to_rva(&addr_map)?.0, symbol.name().ok()?.to_string())),
             _ => None,
         })
-        .collect()
-        .ok()?;
+        .collect()?;
 
     // Add Procedure symbols from the modules, if present. Some of these might
     // duplicate public symbols; in that case, don't overwrite the existing
     // symbol name because usually the public symbol version has the full
     // function signature whereas the procedure symbol only has the function
     // name itself.
     if let Ok(dbi) = pdb.debug_information() {
-        let mut modules = dbi.modules().ok()?;
-        while let Some(module) = modules.next().ok()? {
-            let info = pdb.module_info(&module).ok()?;
-            let mut symbols = info.symbols().ok()?;
-            while let Some(symbol) = symbols.next().ok()? {
+        let mut modules = dbi.modules()?;
+        while let Some(module) = modules.next()? {
+            let info = pdb.module_info(&module)?;
+            let mut symbols = info.symbols()?;
+            while let Some(symbol) = symbols.next()? {
                 if let Ok(SymbolData::Procedure(ProcedureSymbol { offset, .. })) = symbol.parse() {
-                    let name = symbol.name().ok()?;
-                    hashmap
-                        .entry(offset.to_rva(&addr_map)?.0)
-                        .or_insert_with(|| Cow::from(name.to_string().into_owned()));
+                    let name = symbol.name()?;
+                    match offset.to_rva(&addr_map) {
```

Comment:
Let's use `.ok_or_else` as suggested in the other place here, too. And let's use the error string "Failed to map offset to rva using PDB file.".